app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -66,16 +63,6 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
   const [streamUrl, setStreamUrl] = React.useState<string>('');
   const videoRef = React.useRef<HTMLVideoElement>(null);
 
-  React.useEffect(() => {
```

Comment:
Removed as it is causing the error message to show up due to race conditions with authentication/authorization.